experiment/simulation/css/main.css

```diff
@@ -1 +1,72 @@
-/* You CSS goes in here */
\ No newline at end of file
+.page {
+	overflow-x: scroll;
+	overflow-y: scroll;
+}
+
+.but {
```

Comment:
Please rename this class to use a more meaningful name. but doesn't really tell me anything